index.ios.js

```diff
@@ -1,30 +1,33 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- * @flow
- */
 import React, { Component } from 'react';
 import {
   AppRegistry,
   StyleSheet,
   Text,
-  View
+  View,
+  TextInput,
+  TouchableHighlight
 } from 'react-native';
+import TodoList from './src/components/todoList'
 
```

Comment:
Created a class where a stateless component functions would be fine (best practice)